packages/client/src/redux/slices/location-slice.ts

```diff
@@ -1,61 +0,0 @@
-import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
```

Comment:
oops i thought we were just gonna delete the api service but still have a slice for this to hold state (so other components can access current location if needed e.g. post list). maybe i misunderstood tho and someone else can chime in

Reply:
ASHGAJHGDJHF IS IT GONNA BE A SLICE

Reply:
JUST LEAVE IT AS A SLICE IF ITS NOT TOO MUCH WORK OW LGTM

Reply:
I'LL UPDATE TMR